### PR TITLE
Convert DarwinOp to FLU

### PR DESCRIPTION
**Pull request by lukicdarkoo**

### CODE REVIEW

**Review by omichel — DISMISSED** [review]
Thank you.

**Review by omichel — CHANGES_REQUESTED** [review]
Many worlds are broken after this change, like [symmetry.wbt](https://github.com/cyberbotics/webots/blob/develop/projects/robots/robotis/darwin-op/worlds/symmetry.wbt).

**Comment by lukicdarkoo** [comment]
> Many worlds are broken after this change, like [symmetry.wbt](https://github.com/cyberbotics/webots/blob/develop/projects/robots/robotis/darwin-op/worlds/symmetry.wbt).

Oh, sorry, I didn't notice that `RobotisOp2` is derived from `Darwin-op`

**Comment by lukicdarkoo** [comment]
@omichel Let me know if there is any other related world. I think I covered all worlds with the `Darwin-op` and `RobotisOp2`.

File: src/webots/editor/WbProjectRelocationDialog.cpp [hunk 1]
```diff
@@ -422,6 +422,7 @@ void WbProjectRelocationDialog::selectDirectory() {
 }
 
 bool WbProjectRelocationDialog::validateLocation(QWidget *parent, QString &filename, bool isImportingVrml) {
+  return true;
```
Comment (omichel) [inline src/webots/editor/WbProjectRelocationDialog.cpp]: That seems wrong...

**Comment by omichel** [comment]
I believe you missed projects/robots/robotis/darwin-op/worlds/visual_tracking.wbt

**Review by omichel — APPROVED** [review]
OK, it looks good now.

**Pull request merged**